elastio-stream-kafka/consumer.py

```diff
@@ -0,0 +1,62 @@
+from kafka import KafkaConsumer
+import json
+import os
+import argparse
+import logging
+logging.basicConfig(level=logging.CRITICAL)
+
+backup_parser = argparse.ArgumentParser(
+    prog="Elastio stream kafka consumer"
+)
+backup_parser.add_argument("--topic_name", type=str, nargs="?", help="Enter Kafka topic name to backup.")
+backup_parser.add_argument("--brokers", type=str, nargs="+", help="Enter one or more Kafka brokers separated by spaces.")
+args = backup_parser.parse_args()
+bootstrap_servers = args.brokers
+topic_name = args.topic_name
+
+
+consumer = KafkaConsumer(
```

Comment:
Please add a bit more comments to the code, that should be obvious for anyone what's going on here